python/cuml/test/test_dbscan.py

```diff
@@ -60,22 +65,49 @@ def test_dbscan_predict_numpy(datatype):
                  dtype=datatype)
 
     print("Calling fit_predict")
-    cudbscan = cuDBSCAN(eps=3, min_samples=2)
+    handle, stream = get_handle(use_handle)
+    cudbscan = cuDBSCAN(handle=handle, eps=3, min_samples=2)
     cu_labels = cudbscan.fit_predict(gdf)
     skdbscan = skDBSCAN(eps=3, min_samples=2)
     sk_labels = skdbscan.fit_predict(X)
     print(X.shape[0])
     for i in range(X.shape[0]):
         assert cu_labels[i] == sk_labels[i]
+    cudbscan.sync()
```

Comment:
I think this sync is needed before the loop in line 74 starts.

Reply:
Done.